c2rust-transpile/src/translator/assembly.rs

```diff
@@ -17,17 +17,20 @@ enum ArgDirSpec {
     InLateOut,
 }
 
-impl ToString for ArgDirSpec {
-    fn to_string(&self) -> String {
+impl std::fmt::Display for ArgDirSpec {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
```

Comment:
Can you import `Display`, `Formatter`, and `fmt::Result`?

Reply:
Small things like this you can amend commits.  There's no hard rule, though, so use your judgement.  If it seems more substantial you can just add a new commit.  And just try to keep each commit passing CI, which helps with bisecting if we need to do that.